src/plugins/filter/demo/post-filter.js

```diff
@@ -0,0 +1,17 @@
+( function() {
+window[ "wb-filter" ] = {
+	filterCallback: function( $field, $elm, settings ) {
+		var $sections =	$elm.find( "section" ),
```

Comment:
Suggestion:
Include that call back directly in the plugin instead. I think it is common to deal with grouping.

So, instead of targeting directly the ```section``` that could be set through config like:
```html
<div class="wb-filter" data-wb-filter='{"group": "section"}'>
   [...]
</div>
```